### ISSUE TITLE
react 0.14?

### ISSUE THREAD

**Issue by abarroso-pmap** [adazzle/react-data-grid#175]
Is react-data-grid going to be updated for React 0.14? I noticed it locks the React peer dependency to v0.13.3.

**Comment by bakesteve** [comment]
yep. #164 
we're alos looking at pulling in from https://github.com/adazzle/react-data-grid/compare/master...intljusticemission:master so should be there fairly soon.
One note though - we are using tags and matching ourversions to the react version it supports. Its a bit of a pain (and not terribly semver) but does make it easier for folks who need to target specific react versions in other parts of their projects, so will keep v13 support and back port any new features at least 1 version.
So basically, keep an eye on that linked issue

**Comment by abarroso-pmap** [comment]
Thanks @bakesteve!
